pkg/controller/attach_detach.go

```diff
@@ -90,35 +90,37 @@ func (r *ReconcileAttachDetach) Reconcile(ctx context.Context, request reconcile
 		return reconcileReturnOnSuccess(request.Name, r.retryInfo)
 	} else if err != nil {
 		azVolumeAttachment.Name = request.Name
-		return reconcileReturnOnError(azVolumeAttachment, "get", err, r.retryInfo)
+		return reconcileReturnOnError(ctx, azVolumeAttachment, "get", err, r.retryInfo)
 	}
 
+	ctx, workflow := azureutils.GetWorkflowFromObj(ctx, azVolumeAttachment)
+
 	// if underlying cloud operation already in process, skip until operation is completed
 	if isOperationInProcess(azVolumeAttachment) {
-		klog.V(5).Infof("Another operation (%s) is already in process for the AzVolumeAttachment (%s). Will be requeued once complete.", azVolumeAttachment.Status.State, azVolumeAttachment.Name)
+		workflow.Logger().V(5).Infof("Another operation (%s) is already in process for the AzVolumeAttachment (%s). Will be requeued once complete.", azVolumeAttachment.Status.State, azVolumeAttachment.Name)
```

Comment:
If not for this verbose log, we wouldn't need to object the workflow in the `Reconcile` function. Since we expect the reconciler to be invoked for intermediate object updates, is this log ever useful? Let's remove it if not.

Reply:
I agree that this log line is better off removed but what would the rationale be for not objecting the workflow in `Reconcile` function?